### ISSUE TITLE
False positive of `Style/IfUnlessModifier` with tabs indentation

### ISSUE THREAD

**Issue by AlexWayfer** [rubocop-hq/rubocop#7885]
Hello.

I'm trying to update RuboCop to version 0.82.0 for [Flame](https://github.com/AlexWayfer/flame).

I'm using tabs in this project.

And there is a condition which does not fit in one line.

Probably related to #5785

--------

## Expected behavior

No offenses.

## Actual behavior

Offense from `Style/IfUnlessModifier`.

## Steps to reproduce the problem

```ruby
					if !@controller.actions.include?(action) || find_reverse_route(action)
						next
					end
```

![image](https://user-images.githubusercontent.com/6510020/79568789-2a006800-80bf-11ea-91a7-ff4a8286bd1c.png)

## RuboCop version

```
0.82.0 (using Parser 2.7.1.1, running on ruby 2.7.1 x86_64-linux)
```

/cc @DracoAter

**Comment by rrosenblum** [comment]
@AlexWayfer can you please provide some extra information on this.

> And there is a condition which does not fit in one line.

Why does this not fit on one line? Does the correction exceed your configured max line length? I'm not sure how this is related to tab indentation.

**Comment by AlexWayfer** [comment]
> Why does this not fit on one line? Does the correction exceed your configured max line length?

Because the text is larger than max length of line. I don't know how to answer.

Look at the right red arrow, it's wrap guide, 80 chars.

You can copy pasted text and calculate a size.

> I'm not sure how this is related to tab indentation.

Did you see a note about #5785 in the issue description?

Tab is single character, but its size is configurable, and size of tab is not equal size of indentation.

For example: `Layout/IndentationWidth: Width: 1` means "1 tab/space for indentation", and `Layout/IndentationStyle: IndentationWidth: 2` means "tab's size is 2 chars (spaces)".
